src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -41,8 +41,8 @@ public function process(ContainerBuilder $container)
         try {
             $this->container = $container;
             foreach ($container->getDefinitions() as $id => $definition) {
-                if ($definition->isAutowired()) {
-                    $this->completeDefinition($id, $definition);
+                if (false !== $autowired = $definition->getAutowired()) {
```

Comment:
The current adds an extra argument to `completeDefinition` for no reason, it can be done in the method directly, and the check is doing exactly what the method call would do thus ... Up to you.